src/core/plugins/oas31/after-load.js

```diff
@@ -38,6 +39,23 @@ function afterLoad({ fn, getSystem }) {
 
     Object.assign(this.fn, wrappedFns)
   }
+
+  // overrides behavior in OpenAPI 3.1.x, recognizes more intentions
+  const isFileUploadIntended = makeIsFileUploadIntended(getSystem)
+  const { isFileUploadIntended: isFileUploadIntendedWrap, schemaHasType } =
+    wrapOAS31Fn(
+      {
+        isFileUploadIntended,
+        ...(fn.jsonSchema202012 && {
```

Comment:
we're mixing the file upload intentation with unrelated wrapping of schema recognition. Can it have separate block as other assignements?